build.gradle.kts

```diff
@@ -192,12 +189,11 @@ fun Project.setupJvmToolchain() {
     }
 }
 
-fun KotlinMultiplatformExtension.setCompilationOptions() {
-    targets.all {
-        compilations.all {
-            configureCompilation()
-        }
-    }
+subprojects {
+    val compileKotlin = tasks.findByName("compileKotlin") as? KotlinCompilationTask<*>
+        ?: return@subprojects
+
+    compileKotlin.configureCompilerOptions()
```

Comment:
```suggestion
    tasks.withType<KotlinCompilationTask<*>>().configureEach {
        configureCompilerOptions()
    }
```

Reply:
And there is already `subprojects` below, we can reuse it.

Reply:
thanks!